tests/test-admin-deploy-syslinux.sh

```diff
@@ -19,11 +19,31 @@
 
 set -euo pipefail
 
-echo "1..18"
+echo "1..19"
```

Comment:
This needs to be bumped to 20 since the FIFREEZE change. I'm guessing GitHub doesn't complain about it because it's the same hunk change.